test/e2e/apimachinery/garbage_collector.go

```diff
@@ -108,6 +111,14 @@ var (
 	// This timeout covers two resync/retry periods, and should be added to wait timeouts to account for delays
 	// to the GC controller caused by API changes in other tests.
 	gcInformerResyncRetryTimeout = time.Minute
+
+	// Many operations in these tests are per-replica and may require 100 mutating requests. The
+	// default qps of a client is 5. If the qps is saturated, it will take 20s complete 100
```

Comment:
which client is this referring to? are they built-in controllers or the e2e client? do we actually limit those to qps of 5?

Reply:
huh, I see this is just relocating the comment... going spelunking through blame...

Reply:
Looks like the original comment was added in https://github.com/kubernetes/kubernetes/commit/a3f4053cb3a2665400e0d770c47da95008c0fa26

I interpreted this as the the controller QPS. At least that's what it's waiting on where this is used. From what I can tell, most controllers don't set a QPS, so I'm assuming they're just getting the default 5qps from https://github.com/kubernetes/kubernetes/blob/c14e0983fbdfa51ef61bbadf4c364f3a63aa8b75/staging/src/k8s.io/client-go/rest/config.go#L115